### PR TITLE
BioHackathon: Add TGSGapCloser

### PR DESCRIPTION
**Pull request by gallardoalba**

### CODE REVIEW

**Comment by gallardoalba** [comment]
@suchitrathapa

File: tools/tgsgapcloser/macros.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<macros>
+    <token name="@TOOL_VERSION@">1.0.3</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="@TOOL_VERSION@">tgsgapcloser</requirement>
```
Comment (bgruening) [inline tools/tgsgapcloser/macros.xml:6]: Indentation...

**Review by bgruening — APPROVED** [review]

File: tools/tgsgapcloser/macros.xml (hunk 1 above)
Comment (gallardoalba) [reply tools/tgsgapcloser/macros.xml:6]: Sorry, I cannot see the problem in that case, could you provide me more details?

Comment (gallardoalba) [reply tools/tgsgapcloser/macros.xml:6]: I saw it, sorry!

**Comment by bgruening** [comment]
Cool, merci!

Please post that in the slack channel and add it to EU :)

**Pull request merged**